angularjs-google-maps/ng-map.d.ts

```diff
@@ -0,0 +1,148 @@
+﻿// Type definitions for angularjs-google-maps v1.17.3
+// Project: https://github.com/allenhwkim/angularjs-google-maps
+// Definitions by: Niko Kovačič <https://github.com/nkovacic>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../angularjs/angular.d.ts" />
+/// <reference path="../googlemaps/google.maps.d.ts" />
+
+declare module "ngMap" {
```

Comment:
You should use `declare module "ngmap" {`.
https://github.com/allenhwkim/angularjs-google-maps/blob/master/build/scripts/ng-map.js#L3301
this package returns "ngMap", but package name is `ngmap`.